en/advanced_config/esc_calibration.md

```diff
@@ -24,33 +47,90 @@ To calibrate the ESCs:
 1. Remove the propellers.
 
    :::warning
-   Never attempt ESC calibration with props on.
-   
-   The motors should not spin during ESC calibration.
-   However if an ESC doesn't properly support/detect the calibration sequence then it will respond to the PWM input by running the motor at maximum speed.
+   Never attempt ESC calibration with propellers on!
+
+   The motors _should_ not spin during ESC calibration.
+   However if calibration starts when the ESC is already powered, or if the ESC doesn't properly support/detect the calibration sequence, then it will respond to the PWM input by running the motor at maximum speed.
    :::
-1. Disconnect the battery and connect the flight controller via USB (only).
+1. Map the ESCs you're calibrating as motors in the vehicle's [Actuator Configuration](../config/actuators.md).
+   Only mapped actuators get an output, and hence only ESCs mapped as motors will be calibrated.
+
+1. Unpower the ESCs by unplugging the battery.
+   The flight controller should still be powered via USB, and remain connected to the ground station.
+
 1. Open the *QGroundControl* **Settings > Power**, then press the **Calibrate** button.
 
    ![ESC Calibration step 1](../../assets/qgc/setup/esc/qgc_esc_calibration.png)
 
-1. Connect the battery when prompted:
+1. After starting the calibration sequence, power the ESCs (you should be prompted):
+
 
    ![ESC Calibration step 2](../../assets/qgc/setup/esc/esc_calibration_step_2.png)
 
    The calibration will begin automatically:
 
    ![ESC Calibration step 3](../../assets/qgc/setup/esc/esc_calibration_step_3.png)
 
-1. Once the calibration complete you will be prompted to disconnect the battery.
+1. During the calibration you should hear model-specific beeping from the ESC, which indicates the individual steps of the calibration.
+
+   You will be prompted when the calibration completes.
 
+   <a id="actuatorconfig_step"></a>
    ![ESC Calibration step 4](../../assets/qgc/setup/esc/esc_calibration_step_4.png)
 
+1. Go back to the [Actuator Configuration](../config/actuators.md) section.

```

Comment:
I've modified this a bit to remove the parameter names (you shouldn't need to know them when using the actuators config). I've also modified the steps so you use the slider in the current position, rather than messing with the disarmed value and having to set it back afterwards. I just think this is more logical.

NOTE, http://docs.px4.io/main/en/config/actuators.html#actuator-configuration-and-testing is the same process. This has two implications:

1. We should fix up the images/defaults in http://docs.px4.io/main/en/config/actuators.html#actuator-configuration-and-testing to note the new defaults.
2. Should we just link to this section rather than reproducing>"
3. Does http://docs.px4.io/main/en/config/actuators.html#actuator-configuration-and-testing need a bullet point for setting the _maximum_ value as you have here?

Reply:
You are finding the rough spots immediately 👍 

> you shouldn't need to know them when using the actuators config

Completely true for going through the procedure. I only mentioned them for completeness e.g. for people who want to take note and put these settings into an airframe configuration. Not sure if it's appropriate.

> so you use the slider in the current position, rather than messing with the disarmed value

That sounds more logical yes. But I prefer the more hacky-sounding way with the disarmed value. The goal is to find an explicit value by trial and observation and you might realistically end up trying 10 times including all motors. If you do that with the slider the workflow is:
- type a value in the minimum field
- turn the sliders on
- move the slider over the bump
- observe the result and repeat

I'm using the disarmed field because then it's just:
- type a value in the disarmed field
- observe the result and repeat

Additionally, I can't find the maximum value any other way because I do it by listening if the motor spins any faster while changing the command. With the slider workflow I cannot hear the direct difference because the motor gets stopped and spun up again in between tries.

> is the same process

There's especially overlap with https://docs.px4.io/main/en/config/actuators.html#motor-configuration I'm aware this is not ideal. My goal is to have people not skim through the large actuators page and miss the fact that correctly configuring the limits is crucial and in principle more important than the ESC calibration itself.

> Should we just link to this section rather than reproducing

Yes but what's the best way to do that? Making sure users with short attention span go through the important steps with the least amount of confusion. Probably my approach is too local.

> need a bullet point for setting the maximum value as you have here?

Yes, it's part of a bring-up. It usually gets neglected because it's not important for the happy path flying. But ESCs commonly also have a dead zone at the top of the command range. Not configuring it leads to worse performance in corner cases. The controller/allocation relies on that high thrust actuation range being available. If there's no authority for the motor to actually spin faster that's unexpectedly non-linear and tracking gets unpredictably worse whenever that command is hit during flight. Can for example result in wobbles and attitude overshoot when flying with full thrust.

Reply:
You're most welcome!

> so you use the slider in the current position, rather than messing with the disarmed value

Thanks for the explanation. I have rewritten this as you would prefer in https://github.com/PX4/PX4-user_guide/pull/2599/commits/5b748488c917a07fcc2e1756fb8ffd0670b30433 - needs to be checked, and I have a question.

> My goal is to have people not skim through the large actuators page and miss the fact that correctly configuring the limits is crucial and in principle more important than the ESC calibration itself.

I'm fine with reproducing it here - the actuator doc is a monolith. I want to split it a bit, but will probably do so in a following release (a scary job). At that point maybe we revisit whether we want to duplicate this or not.

What we will need now is to update the Actuators doc to show the new defaults for all the images. I guess that will have to wait for your PR to go in?

> need a bullet point for setting the maximum value as you have here?

Great. I added one. Will add a separate discussion point there.